<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>915</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>68</y>
      <width>388</width>
      <height>564</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="Page_Empty"/>
    <widget class="QWidget" name="Page_Homing"/>
    <widget class="QWidget" name="Page_Choose_Object">
     <widget class="QPushButton" name="Moon_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Moon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Jupyter_Button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Jupyter</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Mercury_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Mercury</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sun_Button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sun</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mars_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>420</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Mars</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Saturn_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Saturn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Venus_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Venus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Break_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_Manual_Control">
     <widget class="QPushButton" name="Right_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prawo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Left_Button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lewo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Up_Button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Góra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Down_Button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>460</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dół</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_Other">
     <widget class="QPushButton" name="BT_Connect">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>512</y>
        <width>364</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Combo_Devices">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>64</y>
        <width>364</width>
        <height>436</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertPolicy::InsertAtBottom</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Device_List_Label">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>20</y>
        <width>262</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Device List</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Choose_Object">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>640</y>
      <width>188</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Object</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Other">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>692</y>
      <width>188</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connect Device</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Manual_Control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>692</y>
      <width>188</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manual Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Homing">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>188</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Homing</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
